.travis.yml

```diff
@@ -11,6 +11,7 @@ services:
 
 php:
   - 7.2
+  - 7.3
```

Comment:
the min/max should suffice here

Reply:
I'm not sure I follow this? Travis uses this as the list of versions to test, not a range, and so removing 7.3 would just cause Travis to not test 7.3 (see https://travis-ci.org/MasterOdin/phinx/builds/628789340 as example).

Reply:
Yes, but there is no use case where min/max alone wouldnt suffice.

Reply:
Removed 7.3